kubernetes-services/work-scheduler/test/helpers/env.ts

```diff
@@ -0,0 +1,3 @@
+process.env.SHARED_SECRET_KEY = '_THIS_IS_MY_32_CHARS_SECRET_KEY_';
```

Comment:
Shared secret key was for EDL token passing - why do we need it here?

Reply:
we don't

Reply:
removed